tox.ini

```diff
@@ -51,7 +51,7 @@ deps =
     oldestdeps: sunpy<4.0
     oldestdeps: astropy<4.3
     oldestdeps: gwcs<0.16
-    oldestdeps: numpy<1.18
+    oldestdeps: numpy==1.20.0
```

Comment:
We can't do this yet, as our Numpy version is given by NEP 29. We need to make the import of typing optional.

Reply:
Wouldn't that mean writing 2 different version of the calling sequences on the ABCs based on whether `numpy.typing` is installed or not?

Reply:
Easiest thing to do would be something like this:

```python
try:
    from numpy.typing import NDArrayLike
except ImportError:
    NDArrayLike = Any
```